src/sentry/notifications/notifications/activity/base.py

```diff
@@ -129,12 +129,32 @@ def get_context(self, provider: ExternalProviders | None = None) -> MutableMappi
         should_add_url = provider is not None
         text_description = self.description_as_text(text_template, params, should_add_url, provider)
         html_description = self.description_as_html(html_template or text_template, params)
-        return {
+
+        context = {
             **self.get_base_context(),
             "text_description": text_description,
             "html_description": html_description,
         }
 
+        # Add suspect commits to workflow notifications using the same method as issue alerts
+        if self.group:
```

Comment:
can we add this under a feature flag so we can test internally etc?  https://develop.sentry.dev/backend/application-domains/feature-flags/flagpole